tests/flow/test_graph_merge.py

```diff
@@ -370,7 +370,7 @@ def test19_merge_dependency(self):
         # Starting with an empty graph.
         # Create 2 nodes and connect them to one another.
         self.env.flush()
-        query = """MERGE (a:Person {name: 'a'}) MERGE (b:Person {name: 'b'}) MERGE (a)-[:FRIEND]->(b) MERGE (b)-[:FRIEND]->(a) RETURN a,b"""
+        query = """MERGE (a:Person {name: 'a'}) MERGE (b:Person {name: 'b'}) MERGE (a)-[:FRIEND]->(b) MERGE (b)-[:FRIEND]->(a)"""
```

Comment:
Why change this test?

Reply:
This is something we will need to look into, as currently on my machine the FLUSH-ALL mess the client cache mapping.

Reply:
Strange! Task for another day, though.